lib/rbac_service.rb

```diff
@@ -0,0 +1,48 @@
+require 'rbac-api-client'
+class RBACService
```

Comment:
Same thoughts as above on namespacing.  Though I wonder if a class name of `Request` would be better than service?
`RBAC::Request`, and `RBAC::Access`

Reply:
@syncrou Changed Please review